spec/models/course/requirement/achievement_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
 RSpec.describe Course::Condition::Achievement, type: :model do
-  it { is_expected.to act_as(:condition).class_name(Course::Condition.name) }
+  it { is_expected.to acts_as(:condition).class_name(Course::Condition.name) }
```

Comment:
I used `act_as` because it isn't grammatically correct to say it is expected to acts as. Or is it? I don't know. My England fail.